src/loose/oose/fis/lab/student/manager/Main.java

```diff
@@ -9,13 +9,15 @@
 public class Main extends Application {
 
     @Override
-    public void start(Stage primaryStage) throws Exception{
-        Parent root = FXMLLoader.load(getClass().getResource("sample.fxml"));
-        primaryStage.setTitle("Hello World");
-        primaryStage.setScene(new Scene(root, 300, 275));
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
```

Comment:
suggestion
primaryStage.setTitle("FIS - Student Manager");
```suggestion
        primaryStage.setTitle("Student Manager");
```